python/lsst/obs/pfs/isrTask.py

```diff
@@ -622,6 +660,61 @@ def maskAmplifier(self, exposure, amp, defects):
 
         return badAmp
 
+    def darkCorrectionFromBBoxes(self, bboxes, data, dark, clipPercentiles):
```

Comment:
Since this is a method, should drop `clipPercentiles`, and simply take them from `self.config`.